nss-day-cohort-47/holiday-road-salt-lake-skaters
# Ad save button

# Description

Added functionality so save button is disabled by default and enabled when all three dropdown menus are selected.

Fixes # (issue)

## Type of change

Please delete options that are not relevant.


# Testing Instructions
Load home page. Save button should be disabled. Select items from all three dropdown menus and save button should be enabled.

# Checklist:
